lookout/style/cloner.py

```diff
@@ -0,0 +1,83 @@
+"""Cloning utilities for git repositories."""
+
+import logging
+import os
+from typing import Dict, Optional, Sequence, Tuple
+
+from dulwich import porcelain
+from joblib import delayed, Parallel
+
+
+class ParallelWriteToLogs(Parallel):
```

Comment:
We should use the regular `Parallel` class and remove this.

Reply:
It is a tiny change that does exactly the same thing but has nicer output to the logs and not directly to stdout. why? we put everything in logs but not progress?

Reply:
I thought that joblib was already using logging